docs/dev_guide/feathr_overall_release_guide.md

```diff
@@ -30,11 +30,12 @@ Read through the [commit log](https://github.com/feathr-ai/feathr/commits/main)
 
 ## Code Changes
 Before the release is made, the version needs to be updated in following places
-- [build.sbt](https://github.com/feathr-ai/feathr/blob/main/build.sbt#L3) - For Maven release version
-- [setup.py](https://github.com/feathr-ai/feathr/blob/main/feathr_project/setup.py#L10) - For PyPi release version
-- [conf.py](https://github.com/feathr-ai/feathr/blob/main/feathr_project/docs/conf.py#L27) - For documentation version
+- [build.sbt](https://github.com/linkedin/feathr/blob/main/build.sbt#L3) - For Maven release version
```

Comment:
The org name has been changed from LinkedIn to feathr-ai, this change should get reverted

Reply:
This has been fixed.